MIDAS/src/hardware/Camera.cpp

```diff
@@ -0,0 +1,60 @@
+#include <ACAN2517.h>
+#include <SPI.h>
+
+static const byte MCP2517_SCK  = 0;
+static const byte MCP2517_MOSI = 0;
+static const byte MCP2517_MISO = 0;
+
+static const byte MCP2517_CS  = 0;
+static const byte MCP2517_INT = 0;
+static bool debug = false;
+
+ACAN2517 can (MCP2517_CS, SPI, MCP2517_INT) ;
+
+void setup () {
```

Comment:
the setup function name is reserved for the setup function in main.cpp. you would want to call it something else, like can_setup or something else that makes sense, you can look at the other sensor implementations for ideas